### DISCUSSION TITLE
Refund for multiple purchases

### DISCUSSION THREAD

**Discussion by jonhydetitanbay** [primefaces/community#880] [Figma UI Kit] [answered]
Hi,

We only wanted to buy one licence and not two but we received an error on the purchase screen so we tried again. Turns out we were charged twice.
![Screenshot 2024-01-10 at 10 54 12](https://github.com/primefaces/community/assets/152395829/b9326de8-5331-4501-9ef1-4c3660e16bcc)
Can someone cancel one order and refund us please.

**Comment by mertsincan** [comment, marked as answer]
Hi, 
Please contact us;
contact_at_primetek.com.tr

Best Regards,

**Reply by jonhydetitanbay** [reply]
Great thank you I have done that.
